[WIP] Updated dependencies

truls1502 opened the pull request:
**Updated the packages following:** 
- [x] consolidation/output-formatters
- [x] squizlabs/php_codesniffer
- [x] symfony/finder
- [x] symfony/process
- [x] symfony/yaml
- [ ] symfony/console

truls1502 committed:
Update consolidation/output-formatters from 3.2.1 to 3.4.0

truls1502 committed:
Update squizlabs/php_codesniffer from 2.9.1 to 2.9.2

truls1502 committed:
Update symfony/finder from v3.4.15 to v3.4.18

truls1502 committed:
Update symfony/process from v3.4.15 to v3.4.18

truls1502 committed:
Update symfony/yaml from v3.4.15 to v3.4.18

truls1502 committed:
Update the dependencies to consolidation/robo

truls1502 committed:
Update the dependencies to behat/behat

truls1502 committed:
Update the dependencies to psy/psysh

truls1502 committed:
Update the dependencies to symfony/console

greg-1-anderson reviewed (changes requested):
Please do not pin dependencies in composer.json.  Use `^` notation.

truls1502 committed:
Unpinned consolidation/output-formatters

truls1502 commented:
Hi @greg-1-anderson! I have unpinned `consolidation/output-formatters` after your request.  😊

truls1502 commented:
I managed to update the following commits above.

However, here is something I need your help:
- I had to create a separate PR #1900 for `php-coveralls/php-coveralls` since `satooshi/php-coveralls` is abandoned. _(I just changed the package name)_
- I am unable to update `symfony/console` because it is pinned and the update doesn't work since it failed on Travis. Referring to an unresolved issue #1865

truls1502 committed:
Merge branch 'master' into update/dependencies

violinist-bot committed:
Update consolidation/robo

truls1502 committed:
Update consolidation/robo from 1.3.1 to 1.3.2

truls1502 committed:
Update symfony/yaml from v3.4.15 to v3.4.19

truls1502 committed:
Update symfony/process from v3.4.15 to v3.4.19

truls1502 committed:
Update symfony/finder from v3.4.15 to v3.4.19

truls1502 committed:
Merge branch 'fix/robo' into update/dependencies

truls1502 committed:
Update consolidation/robo dependencies:
Remove g1a/composer-test-scenarios
Update symfony/filesystem from v3.4.18 to v3.4.19
Update symfony/event-dispatcher from v2.8.47 to v3.4.19
Update psr/log from 1.0.2 => 1.1.0
Update consolidation/annotated-command from 2.9.1 to 2.10.0
Update consolidation/robo from 1.3.1to 1.3.2

truls1502 committed:
Update symfony/var-dumper from v3.4.18 to v3.4.19

truls1502 committed:
Update behat/behat dependcies
- Update symfony/class-loader from v3.4.18 to v3.4.19
- Update symfony/config from v3.4.18 to v3.4.19
- Update symfony/dependency-injection from v3.4.18 to v3.4.19
- Update symfony/translation from v3.4.18 to v3.4.19

truls1502 committed:
Update symfony/stopwatch from v3.4.18 to v3.4.19

greg-1-anderson commented:
Note that dependency updates may be done automatically using [dependencies.io](https://www.dependencies.io/docs/dependencies/php/). We do this for [Robo et. al.](https://github.com/consolidation/Robo/blob/master/dependencies.yml).

greg-1-anderson commented:
Hm, I see that at least some of the commits above were generated by a similar bot.

truls1502 commented:
Ok, @greg-1-anderson! So let me know if it is pointless for me to help you by creating this PR to update the dependencies or this PR is not necessary for you. What do you prefer? 😊

truls1502 committed:
Merge branch 'upstream/master' into update/dependencies

truls1502 commented:
I have fixed the lock-file! 😊 Since it has a merge-conflict.

truls1502 committed:
Update guzzlehttp/guzzle dependcies
- Installed ralouphie/getallheaders 2.0.5
- Updated guzzlehttp/psr7 from 1.4.2 to 1.5.2

truls1502 committed:
Update php-coveralls/php-coveralls from v1.1.0 to v2.1.0
- Removed the abandoned package guzzle/guzzle v3.8.1

truls1502 committed:
Updated php-coveralls for travis and composer

truls1502 commented:
I was able to update php-coveralls from version 1 to 2. Please to check it at your end! 😊

truls1502 commented:
Ping to @TeslaDethray and @ronan
